Microsoft.Toolkit.Uwp.SampleApp/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
```

Comment:
I can't run this on my SP4 which was reset last night and with VS and tools reinstalled.